### DISCUSSION TITLE
v1 Readiness

### DISCUSSION THREAD

**Discussion by sarah11918** [withastro/docs#1024] [Ideas]
If there's something about Docs you'd like to see improved before v1, here's your chance!

In particular, let's try to focus on any huge issues and real, achievable improvements we can make in the next 2 weeks or so. Not just a wish list, but what would you suggest we REALLY NEED to tackle before the docs start to get a lot more traffic with the launch announcement??

**Comment by sarah11918** [comment]
Topic 1 - Code sample formatting!

On our Layouts page, we tried to improve the overall look of the code samples. (Specifically with respect to the file name/path).

This is the only page where we've done this: (screenshot) vs all our existing code samples. 

Questions:
1. Do you like the new idea? Would you change anything about it? Even if not this exactly, do you think we can (easily, quickly) do better than the old format?
2. Is it worth the effort to go through and update all our existing code samples before v1?

Some more ideas/context in this issue: https://github.com/withastro/docs/issues/918

![Screenshot 2022-07-18 14 47 51](https://user-images.githubusercontent.com/5098874/179571664-cd1f5c8e-350d-4b8a-a428-f1c2f39a6828.png)

**Comment by ArmandPhilippot** [comment]
I'm closing this since we are more than ready for v1 I think... 🤔 

And 4 years later, we'll be at v7! What a journey for you! 💜
